### PR TITLE
Scrapper, Beryozkin Daniil - 21FPL1

### PR DESCRIPTION
**Pull request by danind**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import requests
```
Comment (demid5111) [inline main.py]: remove all unrelated files

**Review by demid5111 — CHANGES_REQUESTED** [review]
nothing to review. hurry up!

**Review by demid5111 — CHANGES_REQUESTED** [review]
no progress, your work is at risk for the deadline

**Review by demid5111 — CHANGES_REQUESTED** [review]
no progress, your work is at high risk for the deadline

**Comment by demid5111** [comment]
ALERT: transfer of mentor. Now your mentor is @dmitry-uraev

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]
no progress

**Comment by dmitry-uraev** [comment]
no progress

**Comment by dmitry-uraev** [comment]
No progress since last review

**Comment by dmitry-uraev** [comment]
No progress since last review

**Comment by dmitry-uraev** [comment]
Keep up

You should probably increase target score to 6, because you have -2 and that is 2 as for now

File: lab_5_scrapper/scrapper.py [hunk 2]
```diff
@@ -90,25 +191,38 @@ def __init__(self, config: Config) -> None:
...
+        self.urls = []
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        article_href = article_bs.get('href')
+        if isinstance(article_href, str) and article_href.startswith('/doc/'):
+            return f'https://www.kommersant.ru{article_href}'
```
Comment (dmitry-uraev) [inline lab_5_scrapper/scrapper.py]: is there a way to not hardcode the url string?

**Review by dmitry-uraev — CHANGES_REQUESTED** [review]
Some comments for lab 5 and we are done. 

Please, start implementing pipeline.

File: lab_5_scrapper/scrapper.py (hunk 2 above)
Comment (danind) [reply lab_5_scrapper/scrapper.py]: will it be ok to create separate variable inside this function which contains `'https://www.kommersant.ru'` and then return sequence of this variable and article_href?

**Review by dmitry-uraev — COMMENTED** [review]
Make sure you pass all checks till lab handover this Friday

File: lab_5_scrapper/scrapper.py (hunk 2 above)
Comment (dmitry-uraev) [reply lab_5_scrapper/scrapper.py]: instead you can try to get this link dynamically from each url extracted or get this link dynamically from seed_urls at least
